src/main.rs

```diff
@@ -315,11 +316,15 @@ async fn provide_interactive(
 
     let (db, hash) = provider::create_collection(sources).await?;
 
-    println!("Collection: {}\n", Blake3Cid::new(hash));
+    out_writer
```

Comment:
Maybe not?  https://github.com/n0-computer/iroh/pull/790/files#r1115540394

Reply:
lmao you did this to hurt me

😆

Reply:
hey, i only wanted to be able to read "8534363 bytes".  originally i copied you `out_writer` approach until called out on it :sweat_smile: